service/src/main/resources/application.yml

```diff
@@ -0,0 +1,28 @@
+spring:
+  profiles: local-development
+  datasource:
+    url: ${sbac.jdbc.host}/session
+    username: ${sbac.jdbc.user}
+    password: ${sbac.jdbc.password}
+    type: com.zaxxer.hikari.HikariDataSource
+    driver-class-name: com.mysql.jdbc.Driver
+  mvc:
+    throw-exception-if-no-handler-found: true
+  resources:
+    add-mappings: false
+
+flyway:
+  enabled: false
+
+server:
+  port: 8082
+  undertow:
+    buffer-size: 16384
+    buffers-per-region: 20
+    io-threads: 64
+    worker-threads: 512
+    direct-buffers: true
+
+session-service:
+  #Assume we're also running a tds-exam-service in "local-development" mode
+  exam-url: http://localhost:8081
```

Comment:
None of the other micro services do.  TDS_Student does (not student service but the main student application).  Is say just ignore my comment cause I'm probably thinking of something else.

Nothing to see here ... Move along